ui-frontend/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -834,7 +1006,7 @@ module.exports = Backbone.AssociatedModel.extend({
   //
   utmUpstoLL(utmUpsParts) {
     const { hemisphere, zoneNumber, northing } = utmUpsParts
-    const northernHemisphere = hemisphere === 'NORTHERN'
+    const northernHemisphere = hemisphere.toUpperCase() === 'NORTHERN'
```

Comment:
📝  Adds in a bit more safety when it comes to hemisphere case for ease of converting!